### PR TITLE
fix: reload table after action execution

### PR DESCRIPTION
**Pull request by eperedo**
### :pushpin: References

-   **Issue:** Closes https://app.clickup.com/t/865czeue4

### :video_camera: Screenshots/Screen capture

[User-Extended-App.webm](https://github.com/EyeSeeTea/user-extended-app/assets/461124/f6ffd6b8-2247-4e5f-a50a-a91bf68c70a8)

### CODE REVIEW

**Review by ifoche — CHANGES_REQUESTED** [review]
@eperedo not sure why but with this code when you try to replicate a user from table, instead of the classical replicate from table modal, you get the replicate user from template one. Could you please give it a look?

**Comment by eperedo** [comment]
> @eperedo not sure why but with this code when you try to replicate a user from table, instead of the classical replicate from table modal, you get the replicate user from template one. Could you please give it a look?

Sorry my bad. I was using the same name action for both modals. Can you try again?

File: src/webapp/components/user-list-table/UserListTable.tsx [hunk 1]
```diff
@@ -233,15 +231,15 @@ export const UserListTable: React.FC<UserListTableProps> = ({
...
+    }, [openSettings, enableReplicate, editUsers, onReorderColumns, reload, onAction]);
 
     const refreshRows = useCallback(
         async (
             search: string,
             { page, pageSize }: TablePagination,
             sorting: TableSorting<User>
         ): Promise<{ objects: User[]; pager: Pager }> => {
-            console.debug("Reloading", reloadKey);
+            console.debug("Reloading", reloadKey, reloadTableKey);
```
[the remark this answers is not in the document]
Comment (eperedo) [reply src/webapp/components/user-list-table/UserListTable.tsx:254]: I'm using `reloadTableKey` to update the table when actions outside table are executed (old components like replicate from user, set org unit)

File: src/domain/usecases/SaveUserStatusUseCase.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { UseCase } from "../../CompositionRoot";
+import { FutureData, Future } from "../entities/Future";
+import { User } from "../entities/User";
+import { UserRepository } from "../repositories/UserRepository";
+
+export class SaveUserStatusUseCase implements UseCase {
+    constructor(private userRepository: UserRepository) {}
+
+    public execute(users: User[], disabled: boolean): FutureData<void> {
```
Comment (tokland) [inline src/domain/usecases/SaveUserStatusUseCase.ts:9]: best practices: `options: {disabled: boolean}`

**Review by tokland — APPROVED** [review]
All good! Just a minor proposal for an option. Also, can you add .rvmrc with the expected node version?

**Review by MiquelAdell — APPROVED** [review]

**Review by MiquelAdell — APPROVED** [review]

**Pull request merged**